css/hero.css

```diff
@@ -5,7 +5,7 @@
     text-align: center;
     color: white;
     float: left; 
-    margin-top: calc(50vh - 280px);
+    margin-top: calc(50vh - 110px) !important;
```

Comment:
Išmetam !important, to jei norimas stilius nesuveikia:
- perdėlioti CSS failų eiliškumą, jog komponentai būtų žemiau bendrinio stiliaus;
- arba padaryti didesnio specifiškumo selector'ių.